src/client/canvas/svg/text.tsx

```diff
@@ -0,0 +1,34 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+
+import { Transform } from '../../math/transform'
+import { TextGeometry } from '../geometry/text_geometry'
+import { Shape } from '../object/shape'
+
+import { toSvgProps, toSvgTransform } from './svg'
+
+type Props = { model: Shape<TextGeometry>, world: Transform }
+export const Text = observer(({ model: { geometry, appearance }, world }: Props) => {
+  const { x, y, fontFamily, text, textAlign, textBaseline, alignToView } = geometry
+
+  // TODO handle font size and max width properly
```

Comment:
Can you move this comment to the hardcoded `0.015` value, and maybe elaborate a bit. e.g.

`// TODO (Houliston): Calculate fontSize such that the text fits within the given maxWidth`